routes/volunteering_events.js

```diff
@@ -1,7 +1,28 @@
 const Router = require('koa-router');
+const VolunteeringEvent = require('../models/VolunteeringEvent');
+const DBMethods = require('./baseDBMethods');
+const VolunteeringOfferRequestBaseModel = require('../models/VolunteeringOfferRequestBase');
 
 const router = new Router();
 
-// Add sub-routes here using router.get(...), router.post(...), etc.
+function getOfferRequestOfEvent(fieldName) {
+  return async function (ctx) {
+    try {
+      const requestedField = await VolunteeringEvent.findById(ctx.params.id)
+        .populate({ path: fieldName, model: VolunteeringOfferRequestBaseModel });
```

Comment:
I think it'd be more readable if you put the population parameter in separate lines, like:
```
.populate({
  path: fieldName,
  model: VolunteeringOfferRequestBaseModel 
});
```